daisy_workflows/export/export_disk_ext.sh

```diff
@@ -38,10 +31,17 @@ SIZE_GB=$(awk "BEGIN {print int((${SIZE_BYTES}/1000000000)+ 1)}")
 
 echo "GCEExport: Exporting disk of size ${SIZE_GB}GB and format ${FORMAT}."
 
-if ! out=$(qemu-img convert /dev/sdb "/gcs/${GCS_PATH}" -p -O $FORMAT 2>&1); then
+if ! out=$(qemu-img convert /dev/sdb "/gs/${LOCAL_PATH}" -p -O $FORMAT 2>&1); then
   echo "ExportFailed: Failed to export disk source to ${GS_PATH} due to qemu-img error: ${out}"
+  exit
 fi
+echo ${out}
 
+if ! out=$(gsutil cp "/gs/${LOCAL_PATH}" "${GS_PATH}"); then
+  echo "ExportFiled: Failed to copy output image to ${GS_PATH}"
```

Comment:
s/Filed/Failed/
you exit here without printing the gsutil command output, is that intentional?

Reply:
No I missed that part, thanks :P